packages/teleport-publisher-vercel/package.json

```diff
@@ -30,7 +30,7 @@
     "node-fetch": "^2.6.1"
   },
   "devDependencies": {
-    "@types/async-retry": "1.4.3",
+    "@types/async-retry": "^1.3.3",
```

Comment:
We are using `async-retry@1.3.3` but loading types of `1.4.3`. So basically types are not matching and another typescript error catch